.github/workflows/python.yaml

```diff
@@ -0,0 +1,64 @@
+name: Python Features Testing
+on:
+  workflow_call:
+    inputs:
+      python-repo-path:
+        required: true
+        type: string
+        default: 'temporalio/sdk-python'
+      python-repo-ref:
+        required: true
+        type: string
+        default: 'main'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./sdk-python
+    steps:
+      - name: Print git info
+        run: 'echo head_ref: ${{ github.head_ref }}, ref: ${{ github.ref }}, python repo ref: ${{ inputs.python-repo-ref }}'
+        working-directory: '.'
+
+      - name: Checkout SDK features repo
+        uses: actions/checkout@v2
+        with:
+          # Weirdly, this must be specified otherwise if the workflow is re-used from an sdk repo
+          # this will just check out the SDK repo.
+          repository: temporalio/sdk-features
+          path: sdk-features
+      - name: Checkout Python SDK repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ inputs.python-repo-path }}
+          submodules: recursive
+          path: sdk-python
+          ref: ${{ inputs.python-repo-ref }}
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+
+      # Build SDK
+      - uses: Swatinem/rust-cache@v1
```

Comment:
Doesn't this need to be pointed at the right directory? https://github.com/temporalio/sdk-python/blob/25caaf008f50cd446b9c7791ea97f3af3bfe8717/.github/workflows/ci.yml#L33

Reply:
Ah, it does indeed, will fix.

Reply:
Fixed